source/d3d11/draw_call_tracker.cpp

```diff
@@ -1,4 +1,5 @@
 #include "draw_call_tracker.hpp"
+#include "log.hpp"
```

Comment:
Doesn't look like logging is used anywhere in this file, so this line is unnecessary.

Reply:
fixed